File: app/src/main/java/com/akshayaap/mouseremote/Keyboard.java
package com.akshayaap.mouseremote;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageButton;

public class Keyboard extends AppCompatActivity {

    ImageButton imageButton_SwitchToPad;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_keyboard);

        imageButton_SwitchToPad = findViewById(R.id.imageButton_SwitchToPad);



        imageButton_SwitchToPad.setOnClickListener(v->{
            startActivity(new Intent(Keyboard.this, TouchPad.class));
        });
    }
}

File: app/src/main/java/com/akshayaap/touchdroid/ui/fragments/Keyboard.kt
package com.akshayaap.touchdroid.ui.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.akshayaap.touchdroid.R

class Keyboard : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_keyboard, container, false)
    }
}
